app/models/user.rb

```diff
@@ -1,6 +1,9 @@
 class User < ApplicationRecord
 
-  validates_presence_of :name, :address, :city, :state, :zip, :password, :password_confirmation, :role
+  validates_presence_of :name, :address, :city, :state, :zip, :role
+
+  validates :password, :presence => true, :confirmation => true, :on => :create
```

Comment:
I think I understand this, but during a code review, could you explain this further? Or via comments on here?

Reply:
Yeah! Let's can definitely go through this more. The gist is that per user stories (_if_ I'm understanding them correctly), we needed user to be able to update profile info _without_ entering password. So adding this says that password is only required when making a new user, not required upon update.